Rakefile

```diff
@@ -1,4 +1,24 @@
-require "bundler/gem_tasks"
-task :default => :spec
+require 'bundler/gem_tasks'
```

Comment:
@maksimf does this have any implications on our customers build tools?

Reply:
What do you mean? I just changed default rake task not to be `spec`